frontends/mit-open/src/page-components/SearchDisplay/ProfessionalToggle.tsx

```diff
@@ -35,8 +35,11 @@ const StyledRiBriefcase3Line = styled(RiBriefcase3Line)`
 const StyledToggleButton = styled(ToggleButton)(({ theme }) => ({
   width: "100%",
   height: "50px",
+  fontSize: "13px",
   backgroundColor: theme.custom.colors.lightGray2,
   color: theme.custom.colors.darkGray2,
+  borderColor: theme.custom.colors.silverGrayLight,
+
```

Comment:
Typography `subtitle3` and color `darkgray2` (all test that pretty much looks black is  `darkgray2`)

Reply:
Also, text change below (L92) to "View all"

Reply:
I think Ferdi asked for it to be "All" instead of "View All". will make the other changes